Documentation/config.txt

```diff
@@ -2124,12 +2124,12 @@ http.sslBackend::
 	This option is ignored if cURL lacks support for choosing the SSL
 	backend at runtime.
 
-http.schannel.checkRevoke::
+http.schannelCheckRevoke::
 	Used to enforce or disable certificate revocation checks in cURL
 	when http.sslBackend is set to "schannel". Defaults to `true` if
 	unset. Only necessary to disable this if Git consistently errors
 	and the message is about checking the revocation status of a
-	certificate. This option is ignored if cURL lacks support for 
+	certificate. This option is ignored if cURL lacks support for
 	setting the relevant SSL option at runtime.
 
 http.schannel.useSSLCAInfo::
```

Comment:
We need to update this config value too, right?

Reply:
Yes.